client/src/helper/Fetch.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+const server_Url = 'http://localhost:3001';
```

Comment:
this can be saved in an environment variable so that when we push this to production this can be easily changed

Reply:
good point

Reply:
The proxy in package.json file: ` "proxy": "http://localhost:3001"` does work, so we can even skip this part of the url string when making calls to the server.